index.html

```diff
@@ -223,22 +234,28 @@
 Portions of the work on this specification have been funded by the United States
 Department of Homeland Security's (US DHS) Science and Technology Directorate
 under contracts HSHQDC-16-R00012-H-SB2016-1-002, and HSHQDC-17-C-00019, as well
-as  the US DHS Silicon Valley Innovation Program under contracts
-70RSAT20T00000010, 70RSAT20T00000029, 70RSAT20T00000030. The content of this specification does not
-necessarily reflect the position or the policy of the U.S. Government and no
-official endorsement should be inferred.
+as the US DHS Silicon Valley Innovation Program under contracts
+70RSAT20T00000010, 70RSAT20T00000029, 70RSAT20T00000030. The content of this
+specification does not necessarily reflect the position or the policy of the
+U.S. Government and no official endorsement should be inferred.
     </p>
 
     <p>
 Portions of the work on this specification have also been funded by the European
-Union's StandICT.eu program under sub-grantee contract number CALL05/19.
+Union's StandICT.eu program under sub-grantee contract number CALL05/19. The
+content of this specification does not necessarily reflect the position or the
+policy of the European Union and no official endorsement should be inferred.
     </p>
 
     <p>
-Work on this specification has also been supported by the Rebooting the Web of
-Trust community facilitated by Christopher Allen, Shannon Appelcline, Kiara
-Robles, Brian Weller, Betty Dhamers, Kaliya Young, Kim Hamilton Duffy, Manu
-Sporny, Drummond Reed, Joe Andrieu, and Heather Vescent.
+Work on this specification has also been supported by the  <a
+href="https://www.weboftrust.info/">Rebooting the Web of Trust</a> community
+facilitated by Christopher Allen, Shannon Appelcline, Kiara Robles, Brian
+Weller, Betty Dhamers, Kaliya Young, Kim Hamilton Duffy, Manu Sporny, Drummond
+Reed, Joe Andrieu, and Heather Vescent. Development of this specification has
+also been supported by the <a href="https://w3c-ccg.github.io/">W3C Credentials
+Community Group</a>, which has been Chaired by Kim Hamilton Duffy, Joe Andrieu,
+Christopher Allen, Heather Vescent, and Wayne Chang.
```

Comment:
I guess in every case like that, and to avoid unnecessary discussions, it is wise to put the names in alphabetical order...

Reply:
The names are in chronological order, another ordering that's difficult to argue with. :)